srcpkgs/sof-firmware/template

```diff
@@ -1,34 +1,40 @@
 # Template file for 'sof-firmware'
 pkgname=sof-firmware
-version=1.5.1
+version=1.6rc3
```

Comment:
Question, why wouldn't you use https://github.com/thesofproject/sof-bin/tree/stable-v1.6 ?

Reply:
While the branch has the label "stable", there's no tagged release in it, so I'd have to choose the commit at the head of that branch. In general, I assumed it was preferred to choose tagged releases over potentially arbitrary commits.

Reply:
If you can reach out to upstream to ask about proper releases for the stable channels, that'd be awesome!

Reply:
It looks like there's ongoing discussion about it in thesofproject/sof-bin#25; I've left a comment there asking for clarification that this problem will  be fixed for the next release.

Reply:
Heh, that looks messy. Glad they are trying to fix it :)